Dockerfile

```diff
@@ -13,7 +13,7 @@ COPY Pipfile .
 COPY Pipfile.lock .
 
 RUN pip install pipenv
-RUN pipenv install
+RUN pipenv install --pre --clear
```

Comment:
Was the problem with `black` install only local to you or did this happen to others as well? Because I had no troubles with the way it was.

Reply:
I think kiko had older version of Pipfile.lock or so. Anyway, these flags pass diretly to the pipfile lock.